### ISSUE TITLE
Linux issues

### ISSUE THREAD

**Issue by varlesh** [quinton-ashley/nostlan#35]
I like your concept. But now it's impossible use on Linux.
OS KDE Neon (Ubuntu 20.04).
1. Can not run any emulator
For example, dolphin-emu. I create symlink on `emu/wii/dolphin/` to binary `/usr/games/dolphin-emu`
when start game - nothing happens

log
```
/tmp/.mount_nostlagI…ore/launcher.js:100 Uncaught (in promise) TypeError: Cannot read property 'msg4' of undefined
    at Launcher.launch (/tmp/.mount_nostlagI…ore/launcher.js:100)
    at CuiState.onAction (/tmp/.mount_nostlagI…ameMediaSelect.js:9)
    at CUI.doAction (/tmp/.mount_nostlagI…js/contro-ui.js:314)
    at async CUI.buttonPressed (/tmp/.mount_nostlagI…js/contro-ui.js:733)
```

**Comment by varlesh** [comment]
![image](https://user-images.githubusercontent.com/8083855/113578589-16357500-9634-11eb-9299-55d05b111d81.png)

**Comment by quinton-ashley** [comment]
Whoops! This bug was caused because the translations are not complete so I fill in with english but it wasn't working correctly.

**Comment by Albert-davil** [comment]
How to report facebook id, using kali linux, (please tell me)

**Comment by quinton-ashley** [comment]
@Albert-davil I don't know

**Comment by quinton-ashley** [comment]
solved in v1.20.15

**Comment by quinton-ashley** [comment]
Also I'm going to meet up with my Russian friend on Friday and we'll do a full Russian translation! ☺️
